shared/settings/index.js

```diff
@@ -10,7 +12,11 @@ class Settings extends Component {
   static parseRoute () {
     return {
       componentAtTop: {title: 'Settings'},
-      subRoutes: {devMenu},
+      subRoutes: {
+        devMenu,
+        deleteConfirm: DeleteConfirm,
+        removeDevice: RemoveDevice,
```

Comment:
Didn't want to switch the tab to devices for this, so pulled this into the routes here. Does that make sense?